app/gui2/src/stores/graph/index.ts

```diff
@@ -251,10 +251,10 @@ export const useGraphStore = defineStore('graph', () => {
     commitEdit(edit)
   }
 
-  function setNodeContent(id: ExprId, content: string) {
+  function setNodeContent(id: ExprId, content: string, moduleIn?: MutableModule) {
```

Comment:
Is this ever called with a `moduleIn` parameter?

Reply:
Actually, not any more. This was something I was using during debugging. But I think it could be removed again.

Reply:
Removed.